docs/aerodromes/classc/Edinburgh.md

```diff
@@ -4,7 +4,69 @@
 
 --8<-- "includes/abbreviations.md"
 
-Reserved.
+## Positions
 
-!!! note
-    Additional charts to the AIP may be found in the RAAF TERMA document, available towards the bottom of [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
\ No newline at end of file
+| Name            | Callsign           | Frequency  | Login Identifier |
+| --------------- | ------------------ | ---------- | ---------------- |
+| Edinburgh ADC   | Edinburgh Tower    | 118.300    | ED_TWR           |
+| Edinburgh SMC   | Edinburgh Ground   | 134.100    | ED_GND           |
+
+## Airspace
+<figure markdown>
+![EDN ADC Airspace](img/edncircuits.png){ width="700" }
+</figure>
+
+- **ED ADC Airspace**: `SFC` to `A015` within the Edinburgh Control Zone (ED CTR).
+
+## Runway Modes
+Single runway operations only.
+
+## Circuit Procedures
+
+The Edinburgh Circuit Area (EDN CIRA) is active at all times when the Edinburgh CTR is active.
+
+Circuit operations occur within the CTR bounded by the Gawler River to the north and the powerline running from the Gawler River (1.5NM south of the Gawler Township) to the south eastern boundary of the ED CTR. Vertical Limits `SFC` to `A020`
+
+If Parafield RWY 03L is active, it can infringe on the circuit deadside. VFR traffic in the EDN CIRA is instructed to remain on the west of the RWY centreline.
+
+!!! phraseology
+    **EDN ADC**: "CIVIL TRANSIT IN PROGRESS, DEAD-SIDE NOT AVAILABLE"
```

Comment:
Could you please expand on what this phraseology example is for? Is this in the ATIS? Is this directed at individual aircraft or a general broadcast? And is this in response to receiving a boundary ident from PF ADC? Observing traffic operating within X miles of EDN CTR? All operations when RWY03L in use at PF?

Reply:
CIRCUIT EAST PROCEDURES
5.3.1 The tracking tolerance of aircraft departing on the Parafield RWY 03L SID infringes the ECA. To
ensure Edinburgh circuit aircraft and the departing civil aircraft are procedurally separated, VFR aircraft in the
ECA will be required to remain on or west of RWY centreline by use of the phrase:
“CIVIL TRANSIT IN PROGRESS, DEAD-SIDE NOT AVAILABLE”
5.3.2 These restrictions will only be imposed with the concurrence of affected aircraft and will be
removed as soon as possible. Practice PFL / PFO / high key operations from the ECA will not be available
during civil transits.
5.3.3 The circuit east procedure does not preclude recovery to YPED on a visual approach, inclusive of
high key and initial and pit

Reply:
we can just remove that section if you want, probably with the lack of traffic in the aerodrome it might be confusing

Reply:
I've removed it.